[Rewrite] Function Wikis - #2 (#203)
NilPointer-Software/bdfd-wiki: src/bdscript/userJoinedDiscord.md

# $userJoinedDiscord

> Returns the creation date of a given Discord user.

Reviewer: ```suggestion
Returns the account creation date of a given user.
```

> 📌 Besides user IDs, [`$userJoinedDiscord`](./userJoinedDiscord.md) can also return creation date of any valid Discord Snowflake ID. Unlike [`$creationDate`](./creationDate.md), this function allows you to customize the output time format.

## Syntax
```
$userJoinedDiscord[User ID;(Format)]
```

### Parameters

> - `User ID` : The ID of the user whose creation date is to return. 

Reviewer: ```suggestion
- `User ID` : The user whose account creation date will be returned. 
```
